extensions/wikia/AdEngine/js/slot/service/megaAdUnitBuilder.js

```diff
@@ -106,9 +107,9 @@ define('ext.wikia.adEngine.slot.service.megaAdUnitBuilder', [
 	}
 
 	function isMegaSlot(slotName) {
-		return megaSlots.indexOf(slotName) !== -1 ||
-				// TODO move BLB to megaSlots in ADEN-6779
-			(getContextTargeting().skin === 'mercury' && slotName === 'BOTTOM_LEADERBOARD');
+		return slotName === 'BOTTOM_LEADERBOARD' ?
```

Comment:
This if statement is getting bigger. What about:
```
if (context.opts.isBLBMegaEnabled) {
  megaSlots.push('BOTTOM_LEADERBOARD');
}

if (context.opts.isBLBOnMobileEnabled) {
  megaSlots.push('MOBILE_BOTTOM_LEADERBOARD');
}
```
somewhere above?

Reply:
Use `adContext.get('opts.foo')` please

Reply:
and please leave a TODO to cleanup after we fully roll out

Reply:
3x Done.